mongo/client.go

```diff
@@ -747,3 +747,9 @@ func (c *Client) Watch(ctx context.Context, pipeline interface{},
 
 	return newChangeStream(ctx, csConfig, pipeline, opts...)
 }
+
+// SessionsCheckedOut returns the number of sessions checked out from the client's session pool.
+// This method is deprecated and does not have any stability guarantees. It may be removed in the future.
+func (c *Client) SessionsCheckedOut() int {
```

Comment:
I don't love this, but I couldn't think of a better way to do this.

Reply:
I can't think of one either.

Ping @craiggwilson to see if you have any clever ideas.

Reply:
not really... I don't mind having this here. An alternative would be to add an additional client option (during construction) that takes in a variable location with which to keep track of this information or a callback or something, but that's a whole other ball of wax.

Is this just another name for in progress sessions? If so, I'd probably name it something more akin to that and not even mark it as "deprecated." Just make it something users can check if they want.

Reply:
That's fair. We generally use the deprecated/unstable warnings for functions that expose something in the `x` package, which this doesn't. I'll remove the deprecation comment in a new commit.

Reply:
How about `ActiveSessions` or `NumActiveSessions`?

#bikeshed